src/com/owncloud/android/ui/activity/UploadFilesActivity.java

```diff
@@ -72,6 +82,9 @@
 
     private static final String WAIT_DIALOG_TAG = "WAIT";
     private static final String QUERY_TO_MOVE_DIALOG_TAG = "QUERY_TO_MOVE";
+    private static final int CAMERA_REQUEST = 1888;
+    private static final int REQUEST_IMAGE_CAPTURE = 1;
+    private static final int CAPTURED_IMAGE_CHECK = 80;
```

Comment:
Neither this constant

Reply:
I am using REQUEST_IMAGE_CAPTURE but not the other two hence I have deleted them.